### DISCUSSION TITLE
Count the slides

### DISCUSSION THREAD

**Discussion by janstieler** [biati-digital/glightbox#358] [Q&A] [answered]
Hi,
is there a possibility to count all slides?
The slideIndex doesn't help me because its only the index of the  current  slide.

Cheers

**Comment by biati-digital** [comment, marked as answer]
you can try the following.

```javascript
const lightbox = GLightbox({ ...options });
const totalSlides = lightbox.elements.length;
```

**Reply by janstieler** [reply]
A thanks. I tried `lightbox.length;` but obvious it can't work.
